model/clusters_mgmt/v1/cluster_operators_type.model

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Metric included in a dashboard.
+struct ClusterOperatorMetric {
+
+    Time Date
+
+	// Name of the metric.
```

Comment:
This is not the name of the metric, but the name of the cluster operator.

Would also be nice to have some documentation in the other attributes explaining what they are, even if it is a single sentence.

@cben would you be so kind to explain what are the meanings of these attributes?

Reply:
sorry for delay.  sure, I'll send a followup.